h2o-automl/src/main/java/ai/h2o/automl/Leaderboard.java

```diff
@@ -732,7 +731,11 @@ public TwoDimTable toTwoDimTable(String tableHeader, boolean leftJustifyModelIds
     //long[] timestamps = getTimestamps(models);
     String[] modelIDsFormatted = new String[models.length];
 
-    if(models[0]._output.isBinomialClassifier()) {
```

Comment:
Can we tell if we are working on a binomial problem without having the first model?  Would be more consistent.

Reply:
Not sure. We could before when we just had a single sort metric for each model type. But now a user can pass in any.

Reply:
It seems like each model makes the decision what family to use itself AutoML doesn't really know what problem is being solved based on the code. In that case this is fine.